app/actions/websocket/posts.ts

```diff
@@ -264,8 +264,10 @@ export async function handlePostUnread(serverUrl: string, msg: WebSocketMessage)
 
     const myChannel = await getMyChannel(database, channelId);
     if (!myChannel?.manuallyUnread) {
-        // We used to fetch the channel to get the delta on the message count
-        // We just need to identify that there are unread messages, the exact number is uninportant
-        markChannelAsUnread(serverUrl, channelId, msgCount, mentionCount, lastViewedAt);
+        const {channels} = await fetchMyChannel(serverUrl, teamId, channelId, true);
+        const channel = channels?.[0];
+        const postNumber = channel?.total_msg_count;
+        const delta = postNumber ? postNumber - msgCount : msgCount;
```

Comment:
Out of curiosity, can `msg_count` be undefined?

Reply:
no, it is received as part of the WS event.. not undefined and not 0